ucm2/sof-soundwire/RT1308-1.conf

```diff
@@ -0,0 +1,31 @@
+# Use case Configuration for sof-soundwire card
+
+If.RT1308-1 {
+	Condition {
+		Type String
+		Haystack "${CardComponents}"
+        	Needle "spk:rt1308" 
```

Comment:
I didn't find the definition of "spk:rt1308" of the components in the source code. Maybe I missed something?